e2e/cypress/integration/notifications/desktop_notifications_spec.js

```diff
@@ -56,6 +56,58 @@ describe('Desktop notifications', () => {
         });
     });
 
+    it('MM-T487 Desktop Notifications - For all activity with apostrophe, emoji, and markdown in notification', () => {
+        cy.apiCreateUser({}).then(({user}) => {
```

Comment:
nit: this could also be
```suggestion
        cy.apiCreateUser().then(({user}) => {
```